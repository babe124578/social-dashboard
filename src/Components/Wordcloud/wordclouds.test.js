import React from 'react';
import Wordclouds from "./wordclouds.js";
import { shallow } from 'enzyme';
import shallowRenderer from 'react-test-renderer/shallow'

it('shallow snapshot', () => {
    const renderer = new shallowRenderer()
    const snapshot = renderer.render(<Wordclouds />)
    expect(snapshot).toMatchSnapshot()
});

it('should render without crashing', () => {
    shallow(<Wordclouds />);
});