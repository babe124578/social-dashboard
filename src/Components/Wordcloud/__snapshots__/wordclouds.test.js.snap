// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shallow snapshot 1`] = `
<div
  style={
    Object {
      "height": 300,
      "width": 500,
    }
  }
>
  <ReactWordCloud
    callbacks={
      Object {
        "getWordTooltip": [Function],
      }
    }
    maxWords={100}
    minSize={
      Array [
        300,
        300,
      ]
    }
    options={
      Object {
        "deterministic": false,
        "enableTooltip": true,
        "fontFamily": "impact",
        "fontSizes": Array [
          5,
          60,
        ],
        "fontStyle": "normal",
        "fontWeight": "normal",
        "padding": 1,
        "rotationAngles": Array [
          0,
          90,
        ],
        "rotations": 1,
        "scale": "sqrt",
        "spiral": "archimedean",
        "transitionDuration": 0,
      }
    }
    words={
      Array [
        Object {
          "text": "WordCloud",
          "value": 13000,
        },
      ]
    }
  />
</div>
`;
